### PR TITLE
Main readme modifications for v0.1.0

### PR DESCRIPTION
**Pull request by adizere**
Closes: #574 
[Rendered](https://github.com/informalsystems/ibc-rs/blob/adi/v010_readme/README.md)

There are two todos left:
- [ ] correct links to crates.io and docs.io for relayer & relayer-cli
- [x] Link to the relayer guide

## Description

______

For contributor use:

- [ ] Updated the __Unreleased__ section of [CHANGELOG.md](https://github.com/informalsystems/ibc-rs/blob/master/CHANGELOG.md) with the issue.
- [ ] If applicable: Unit tests written, added test to CI.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,33 @@
...
+> TODO: update the crate and docs links below (for relayer and relayer-cli).
 
-    ```bash
-    ibc-rs $ cargo run --bin hermes -- -c ./relayer/tests/config/fixtures/relayer_conf_example.toml start --reset
-    ```
+| Crate name    |   Type   |     Version       | Docs |
+| ------------- |:------:|:-------------:| -----:|
+| [ibc (modules)](./modules) | lib|  [![IBC Crate][ibc-crate-image]][ibc-crate-link] | [![Docs][ibc-docs-image]][ibc-docs-link] |
+| [relayer](./relayer)      | lib |  [![Relayer Crate][relayer-crate-image]][relayer-crate-link]  | [![Docs][relayer-docs-image]][relayer-docs-link] |
+| [relayer-cli](./relayer-cli)  | bin: [hermes](relayer-cli/) |  [![Relayer CLI Crate][relayer-cli-crate-image]][relayer-cli-crate-link]      |  [![Docs][relayer-cli-docs-image]][relayer-cli-docs-link] |
```
Comment (romac) [inline README.md:20]:
```suggestion
| [ibc-relayer](./relayer)      | lib |  [![Relayer Crate][relayer-crate-image]][relayer-crate-link]  | [![Docs][relayer-docs-image]][relayer-docs-link] |
| [ibc-relayer-cli](./relayer-cli)  | bin: [hermes](relayer-cli/) |  [![Relayer CLI Crate][relayer-cli-crate-image]][relayer-cli-crate-link]      |  [![Docs][relayer-cli-docs-image]][relayer-cli-docs-link] |
```

File: README.md [hunk 2]
```diff
@@ -1,100 +1,33 @@
...
-- [`gaia`](https://github.com/cosmos/gaia), a blockchain supporting IBC 
-
-Clone the relayer implementation from [cosmos/relayer](https://github.com/cosmos/relayer/).
-We are interested in two commands we can run from this repo:
-
-- `bash scripts/two-chainz "local" "skip"`. Running this script will instantiate two chains, listening on ports `26557` and `26657`, respectively.
+[![Build Status][build-image]][build-link]
+[![End to End testing][e2e-image]][e2e-link]
+[![Apache 2.0 Licensed][license-image]][license-link]
+![Rust Stable][rustc-image]
```
Comment (romac) [inline README.md:6]:
```suggestion
![Rust Stable][rustc-image]
![Rust 1.49+][rustc-version]
```

File: README.md [hunk 3]
```diff
@@ -128,3 +61,24 @@ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
...
+[relayer-cli-docs-image]: https://docs.rs/ibc/badge.svg
+[relayer-cli-docs-link]: https://docs.rs/ibc/
+
+[build-image]: https://github.com/informalsystems/ibc-rs/workflows/Rust/badge.svg
+[build-link]: https://github.com/informalsystems/ibc-rs/actions?query=workflow%3ARust
+[e2e-image]: https://github.com/informalsystems/ibc-rs/workflows/End%20to%20End%20testing/badge.svg
+[e2e-link]: https://github.com/informalsystems/ibc-rs/actions?query=workflow%3A%22End+to+End+testing%22
+[license-image]: https://img.shields.io/badge/license-Apache2.0-blue.svg
+[license-link]: https://github.com/informalsystems/ibc-rs/blob/master/LICENSE
+[rustc-image]: https://img.shields.io/badge/rustc-stable-blue.svg
```
Comment (romac) [inline README.md:84]:
```suggestion
[rustc-image]: https://img.shields.io/badge/rustc-stable-blue.svg
[rustc-version]: https://img.shields.io/badge/rustc-1.49+-blue.svg
```

File: README.md [hunk 4]
```diff
@@ -128,3 +64,25 @@ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
...
+[relayer-cli-crate-image]: https://img.shields.io/crates/v/ibc.svg
+[relayer-cli-crate-link]: https://crates.io/crates/ibc
+[relayer-cli-docs-image]: https://docs.rs/ibc/badge.svg
+[relayer-cli-docs-link]: https://docs.rs/ibc/
+
+[build-image]: https://github.com/informalsystems/ibc-rs/workflows/Rust/badge.svg
+[build-link]: https://github.com/informalsystems/ibc-rs/actions?query=workflow%3ARust
+[e2e-image]: https://github.com/informalsystems/ibc-rs/workflows/End%20to%20End%20testing/badge.svg
+[e2e-link]: https://github.com/informalsystems/ibc-rs/actions?query=workflow%3A%22End+to+End+testing%22
+[license-image]: https://img.shields.io/badge/license-Apache2.0-blue.svg
```
Comment (romac) [inline README.md:85]:
```suggestion
[license-image]: https://img.shields.io/badge/license-Apache_2.0-blue.svg
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -56,8 +54,7 @@ of smaller incremental changes, in the form of small PRs that can be merged
 quickly. This helps manage the load for reviewers and reduces the likelihood
 that PRs will sit open for longer.
 
-![Contributing
-flow](https://github.com/tendermint/tendermint/blob/v0.33.6/docs/imgs/contributing.png)
+![Contributing flow](https://github.com/tendermint/tendermint/blob/v0.33.6/docs/imgs/contributing.png?raw=true)
```
Comment (romac) [inline CONTRIBUTING.md:57]: 👍

File: README.md [hunk 6]
```diff
@@ -1,100 +1,36 @@
...
-We are interested in two commands we can run from this repo:
-
-- `bash scripts/two-chainz "local" "skip"`. Running this script will instantiate two chains, listening on ports `26557` and `26657`, respectively.
+[![Build Status][build-image]][build-link]
+[![End to End testing][e2e-image]][e2e-link]
+[![Apache 2.0 Licensed][license-image]][license-link]
+![Rust Stable][rustc-image]
+![Rust 1.49+][rustc-version]
 
+Rust implementation of Interblockchain Communication (IBC) **modules** and
```
Comment (romac) [inline README.md:9]:
```suggestion
Rust implementation of Inter-Blockchain Communication (IBC) **modules** and 
```

For consistency with https://cosmos.network/ibc

File: README.md [hunk 7]
```diff
@@ -1,100 +1,36 @@
...
+[![Build Status][build-image]][build-link]
+[![End to End testing][e2e-image]][e2e-link]
+[![Apache 2.0 Licensed][license-image]][license-link]
+![Rust Stable][rustc-image]
+![Rust 1.49+][rustc-version]
 
+Rust implementation of Interblockchain Communication (IBC) **modules** and 
+**relayer**. The relayer is split into two crates: a library, and the 
+relayer CLI comprising the main binary, called `hermes`.
+See the table below.
```
Comment (romac) [inline README.md:12]:
```suggestion
Rust implementation of the Inter-Blockchain Communication (IBC) protocol.

This project consists of the following crates:
 
 - The [`ibc`](https://docs.rs/ibc) crate defines the main data structures and on-chain logic  for the IBC protocol.
- The [`ibc-relayer`](https://docs.rs/ibc-relayer) crate provides an implementation of an IBC relayer, as a library.
- [`hermes`](https://hermes.informal.systems) is a command-line interface for the IBC relayer provided by this project.

See the table below for details.
```

**Review by ancazamfir — APPROVED** [review]
Looks good! Thanks Adi!

**Pull request merged**